src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
+
+    protected double radius;
```

Comment:
Parent class should contain common fields of child classes.
That's why color is correct, but radius is property of Circle only so should be there
```suggestion
```